### ISSUE TITLE
[EDH-2751] Search for technical user and agreement 'valid from' and 'valid to' is not working right / notMVP

### ISSUE THREAD

**Issue by kirkekrusell** [Elering/estfeed-datahub-docs#52]
The searches based on validity for technical user and agreements are not working right.

This is because right now the search works like that:

'Valid from':
    changes the search result so that the "Valid to" date would be later than the date entered in 'Valid form' cell
'Valid to':
    changes the search result so that the "Valid from" date would be before the date entered in 'Valid to' cell

Basically the date search is switched.

**Comment by jeksatr** [comment]
In Live since 10.04.2025 08:30
